monetio/models/raqms.py

```diff
@@ -0,0 +1,111 @@
+# Reader for RAQMS real-time files.
+
+import xarray as xr
+
+
+def open_dataset(fname):
+    """Open a single dataset from RAQMS output. Currently set for netcdf file format.
+    Parameters
+    ----------
+    fname : string
+        Filename to be opened.
+    Returns
+    -------
+    xarray.Dataset
+    """
+    names, netcdf = _ensure_mfdataset_filenames(fname)
+    try:
+        if netcdf:
+            f = xr.open_dataset(names[0], drop_variables=["theta"])
+            f = _fix_grid(f)
+            f = _fix_time(f)
+        else:
+            raise ValueError
+    except ValueError:
+        print(
+            """File format not recognized. Note that files should be
+                preprocessed to netcdf."""
+        )
```

Comment:
```suggestion
        print(
            "File format not recognized. Note that files should be "
            "preprocessed to netcdf."
        )
```
otherwise unintended indentation